content/handbook/product/cross-stage-features/merge-requests.md

```
@@ -0,0 +1,103 @@
+
+**マージリクエスト (MR) は、GitLab におけるユーザー間のコラボレーションの中心点です。** ここでコードがレビューされ、パイプラインが実行され、テストおよびスキャンの結果が提供され、最終的にコントリビューションがマージされます。
+
+MR 体験に対する変更は、たとえ小さな変更であっても、**コードレビューの所要時間** に影響を与える可能性があります。コードレビュー時間の短縮は、[Code Review グループの主要な成功指標](https://about.gitlab.com/direction/create/code_review_workflow/#metrics-of-success) です。この指標は、GitLab のすべてのユーザーのコード品質と速度に影響します。
+
+ここ [Code Review グループ](/handbook/product/categories/#code-review-group) では、MR 体験を深く大切にしています。多くの他のグループも MR 体験に利害関係を持ちますが、私たちはその主要なコントリビューターです。**長年にわたり、ガイダンスやビジョンなしに異なる関係者からなされたコントリビューションが、大量の技術的負債と先送りされた UX を生み出してきました**。このページでは、「誰でもコントリビュートできる」精神を持続可能な形で MR 体験に反映させるための、待望のガイダンスを提供します。
+
+## MR 体験にコントリビュートしたいですか?
+
+それは素晴らしいことです! 私たちは他のグループがコントリビュートしてくれるのを歓迎しています。このページの使い方を以下のステップで簡単に説明します:
+
+1. まだ読んでいない場合は、上記の [背景](#background) セクションを読んでください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)